nomenclature/__init__.py

```diff
@@ -93,10 +93,8 @@ def validate(df):
         Return `True` if all column entries in `df` are valid
         or `False` otherwise
     """
-    if not isinstance(df, IamDataFrame):
-        df = IamDataFrame(df)
+    df = IamDataFrame(df)
     success = True
-
```

Comment:
I'd prefer to keep this empty line to better structure the code...

Reply:
Yes, that is true.